.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,2 +1,19 @@
-By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
+### What does this PR do / why do we need it?
 
+### Which issue(s) does this PR fix?
+
+Fixes #
+
+### Special notes for the reviewer:
+
+### Does this PR require changes to documentation?
+
+### All Submissions:
+
+* [ ] Have you followed the guidelines in our Contributing document?
+* [ ] Have you written or refactored unit tests to cover the change?
+* [ ] Have you ran all unit tests and ensured they are passing?
+* [ ] Have you well smoke tested each feature that is being added/modified?
```

Comment:
nit: "well smoke tested" doesn't make sense to me (might be a dialect thing?). Consider "Have you manually tested each feature that is being added/modified?".